CHANGELOG.md

```diff
@@ -21,6 +22,8 @@
 * The storage overrides for configuring per-broker storage class are not supported anymore.
   If you are using the storage overrides, you should instead use multiple KafkaNodePool resources with a different storage class each.
   For more details about migrating from storage overrides, please follow the [documentation](https://strimzi.io/docs/operators/0.45.0/full/deploying.html#con-config-storage-zookeeper-str).
+* Because of the corresponding configuration changes, the 0.32.0 is the minimum HTTP bridge release working with the Strimzi operator 0.46.0.
+  * If you have an Apache Kafka cluster running together with an HTTP bridge instance, please upgrade the bridge after upgrading the operator.
```

Comment:
I think this part is confusing -> the Bridge has no separate versioning when deployed through the operator and there is no way to upgrade the Bridge after upgrading the operator. For most users, all of this happens automatically. This might be relevant only to someone with heavily customized operator installation. I wonder if the previous line is sufficient for that and those using customized installation would understand it. But if you think that is not sufficient, we definitely need to clarify this part to not confuse anyone.

Reply:
You are right. You can't upgrade the bridge, it's coming with the operator installation. I will remove this sentence.